doc/api/fs.md

```diff
@@ -3266,9 +3262,8 @@ The callback will be given three arguments `(err, bytesWritten, buffer)` where
 If this method is invoked as its [`util.promisify()`][]ed version, it returns
 a `Promise` for an `Object` with `bytesWritten` and `buffer` properties.
 
-Note that it is unsafe to use `fs.write` multiple times on the same file
-without waiting for the callback. For this scenario,
-`fs.createWriteStream` is strongly recommended.
+It is unsafe to use `fs.write` multiple times on the same file without waiting
```

Comment:
`` `fs.write` `` -> `` `fs.write()` ``?

Reply:
This and the other similar nit below addressed. Thanks!